test/stress_tests.py

```diff
@@ -197,7 +197,7 @@ def g(x):
 def ray_start_reconstruction(request):
     num_nodes = request.param
 
-    plasma_store_memory = 10**9
+    plasma_store_memory = int(0.5 * 10**9)
```

Comment:
This change was needed because the space calculations in plasma are now done slightly differently (there can now be a discrepancy between the plasma store memory and the size of the memory mapped files, in practice I have seen < 10%).

This change is to be on the safe side to prevent flakiness. With 0.8 here I was still seeing test_simple and test_recursive in stress_tests.py failing occasionally.